torchci/pages/testing_overhead.tsx

```diff
@@ -0,0 +1,46 @@
+import { Grid } from "@mui/material";
+import TimeSeriesPanel from "components/metrics/panels/TimeSeriesPanel";
+import dayjs from "dayjs";
+import { RocksetParam } from "lib/rockset";
+import { useState } from "react";
+
+const ROW_HEIGHT = 240;
+
+export default function TestingOverhead() {
+    // Looking at data from the past six months
+    const [startTime, setStartTime] = useState(dayjs().subtract(1, 'month'));
+
+    const timeParams: RocksetParam[] = [
+        {
+        name: "startTime",
+        type: "string",
+        value: startTime,
+        }
+    ];
+
+    return (
+        <><Grid container spacing={1}>
+            <Grid item xs={24} lg={12} height={ROW_HEIGHT}>
+                <TimeSeriesPanel
+                    title={"Average Time for Workflow (excluding unstable and inductor)"}
+                    queryName={"test_times_per_workflow_type"}
+                    queryCollection={"commons"}
+                    queryParams={[
+                        {
```

Comment:
Hmm, I don't think you use this `workflow_type` param in the query.  On the other hand, you could have an excluded_list param to include unstable and inductor, i.e. passing `excluded_workflows` as a string variable `unstable,inductor` and on Rockset:

```
NOT ARRAY_CONTAINS(SPLIT(:excluded_workflows, ','), LOWER(workflow_name))
```

Reply:
It's used. A seperate table is generated for pull, trunk, and periodic.